test/object.civet

```diff
@@ -1408,9 +1408,9 @@ describe "object", ->
     testCase """
       with reserved word keys
       ---
-      state.{and,await,break,case,catch,class,const,continue,debugger,default,delete,do,else,enum,export,extends}
+      state.{and,await,break,case,catch,class,const,continue,debugger,default,delete,else,enum,export,extends}
       ---
-      ({and:state.and,await:state.await,break:state.break,case:state.case,catch:state.catch,class:state.class,const:state.const,continue:state.continue,debugger:state.debugger,default:state.default,delete:state.delete,do:state.do,else:state.else,enum:state.enum,export:state.export,extends:state.extends})
+      ({and:state.and,await:state.await,break:state.break,case:state.case,catch:state.catch,class:state.class,const:state.const,continue:state.continue,debugger:state.debugger,default:state.default,delete:state.delete,else:state.else,enum:state.enum,export:state.export,extends:state.extends})
```

Comment:
Oh, I didn't realize that it's property globs that are failing. This is annoying. But fixing it will require some work on `PropertyGlob` to not use `BracedObjectLiteral`, but rather just use a subset of those rules. I can work on this in another PR.

Reply:
Thanks for clarifying. I do not know much about property globs so I'll safely leave it with you.

Reply:
I was able to fix this without messing with globs; just forbidding implicit blocks with `do` and `loop`.